ansible/roles/ci/doc/setup/shippable.md

```diff
@@ -0,0 +1,26 @@
+# Shippable Server
+
+## How to setup
+
+The following repository can be used as an example of [Shippable](https://wwww.shippable.com/) configuration [build servers check](https://github.com/shadow-robot/build-servers-check)
+
+The process of setting up [Shippable](https://wwww.shippable.com/) server
+
+  * In the root folder of your repository
+    * Copy *shippable.yml* from [build servers check](https://github.com/shadow-robot/build-servers-check) repository. 
+    * Remove *env* section and *after_failure* sections.
+    * Create *repository.rosinstall* in case if you want to install some packages from source code. This file should be in in [rosinstall](http://wiki.ros.org/rosinstall) format.
+      You can use *repository.rosinstall* [build servers check](https://github.com/shadow-robot/build-servers-check) as an example.
+  * *shippable.yml* contains list of the modules which can be used in the **used_modules** variable. It can be adjusted to any amount of the [modules needed](../modules.md).
```

Comment:
*shippable.yml* contains a list of the modules which can be used in the **used_modules** variable. It can be adjusted to any amount of [modules needed](../modules.md).